### ISSUE TITLE
Glitches after finishing Game

### ISSUE THREAD

**Issue by Gimmo** [CigarProject/Cigar#386]
After the game is finished, screen under overlay repeat last motion. If you move right before - it continue motion , but not uniform: it makes small (greed size?) moves back with time interval about 1 sec.
If last operation you made was zoom - it will make small smooth zoom and then jumping zoomout. And again,and again.

It's very annoying. 
When I press Escape key one more time, motion stops.

I tried with a few servers (mostly MultiOgar 1.2.69) on last Chrome win7/64

**Comment by MastaCoder** [comment]
This is due to serverTimeout I think. Try to change the amount of time it takes for the server to timeout and kill cells or connections. Can you confirm if this fixes it?

**Comment by Gimmo** [comment]
I have not access to server. It belongs to other people.  But the same server with agar.io official client doesn't have this effect. I find another server ( the same version ), which have not this effect. One of 5 i try.

P.S. You can see this effect right after connectioh with server - gameboard image  shows something like heartbeat.

**Comment by MastaCoder** [comment]
Yeah I've seen this, I think it's to do with the server being disconnected and connected again constantly after the death. Something is resetting the connection, probably has to do with the client ending the connection or something. I'll look into it.

**Comment by Gimmo** [comment]
I think there is nothing bad with reconnections, but looks like some initial values (as zoom) on client is different to the default values server send.  So, the problem lies in initial procedure after connection IMO.

**Comment by MastaCoder** [comment]
Probably, but it's weird it does that.

**Comment by Gimmo** [comment]
In Spectate mode client change zoom and jumps around the gamefield  on it's own reasons.
